content/posts/mobile_app_phase_one_report.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Mobile App: Phase One Report"
+date: 2020-07-09T08:51:49+05:30
+draft: false
+author: Nitish Aggarwal ([Nitish145](https://github.com/Nitish145))
+---
+
+The first evaluations happened from June 29th to July 3rd and Yay.. I Passed!!! This blog post comes as a summary to the amazing work done in the first phase of Google Summer of Code.
+
+<p align="center">
+	<img src="/img/first_eval_dashboard.png" alt = "First Evaluations Passed">
+</p>
+
+The mobile_app project was primarily focussed on two sub projects as follows:
+
+1. CircuitVerse API development.
+2. The Mobile App.
+
+## Overview
+
+Before the GSoC period CircuitVerse was primarily focussed for the Web. We have this very amazing [simulator](http://circuitverse.org/simulator), ability to create groups, beautiful circuits, assignments with grading obviously <3 and a lot more features. make sure to check them out on our [website](https://circuitverse.org/). Thanks to our amazing community, [@satu0king](https://github.com/satu0king), [@tachyons](http://github.com/tachyons/) & [@nitin10s](https://github.com/nitin10s).
+
+CircuitVerse now plans to expand.. GSoC 2020 basically aims at improving the Simulator and Web UI, structuring & refactoring the huge JS codebase, offline electron app, the mobile_app and implementing API.
+
+<br/>
+
+## The CircuitVerse API
+
+### Authentication & Users API
+
+For authenticating users, token auth is being used to `encode` user data and return the token for successful login/signup. The `private_key.pem` can also be fetched in `/private_key.pem` which could be used to `decode` the token to fetch user info on client side. Also added `/password/forgot` to send reset password instructions. Coming onto users, all users public info is made available at `/users` & logged in user at `/me`.
```

Comment:
public_key.pem, not private_key.pem

Reply:
Public key is not needed for decoding the token, it is only for validating the token signature

Reply:
Oops! My bad, will updates these.